File: app/src/main/java/com/example/sig/Objects/Report.java
package com.example.sig.Objects;

public class Report {
    private int id_report;
    private int id_user;
    private int id_object;
    private String description;

    public Report() {
    }

    public int getId_report() {
        return id_report;
    }

    public void setId_report(int id_report) {
        this.id_report = id_report;
    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public int getId_object() {
        return id_object;
    }

    public void setId_object(int id_object) {
        this.id_object = id_object;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: application/modules/ws/models/Report.php
<?php

class Report {
    private $_id_report;
    private $_id_user;
    private $_id_object;
    private $_description;

    //CONSTRUCTEURS
    public function __construct()
    {
        $ctp = func_num_args();
        $args = func_get_args();
        switch($ctp)
        {
            case 3:
                $this->_id_report = 0;
                $this->_id_user = $args[0];
                $this->_id_object = $args[1];
                $this->_description = $args[2];
            break;
            case 1:
                $this->hydrate($args[0]);
            break;
            default:
                break;
        }
    }

    //HYDRATATION
    public function hydrate(array $data)
    {
        foreach ($data as $key=>$value){
            $method='set'.ucfirst($key);
            if(method_exists($this,$method)){
                $this->$method($value);
            }
        }
    }

    /**
     * @return mixed
     */
    public function getId_Report(){ return $this->_id_report; }

    /**
     * @param mixed $id_report
     */
    public function setId_Report($id_report){ $this->_id_report = $id_report; }

    /**
     * @return mixed
     */
    public function getId_User(){ return $this->_id_user;}

    /**
     * @param mixed $id_user
     */
    public function setId_User($id_user){ $this->_id_user = $id_user; }

    /**
     * @return mixed
     */
    public function getId_Object(){ return $this->_id_object; }

    /**
     * @param mixed $id_object
     */
    public function setId_Object($id_object){ $this->_id_object = $id_object; }

    /**
     * @return mixed
     */
    public function getDescription(){ return $this->_description; }

    /**
     * @param mixed $description
     */
    public function setDescription($description){ $this->_description = $description; }
}